vhost-user-backend/src/backend.rs

```diff
@@ -59,6 +59,9 @@ pub trait VhostUserBackend: Send + Sync {
     /// Get available vhost protocol features.
     fn protocol_features(&self) -> VhostUserProtocolFeatures;
 
+    /// Reset the emulated device state.
```

Comment:
small nitpick: I think you should also add documentation comment that this is a default implementation.

Reply:
Sure!